apidocs.yaml

```diff
@@ -205,6 +205,7 @@ paths:
                   "clientId" : 1234,
                   "attributes": {
```

Comment:
Do we have this attributes array? if not remove this from doc.

Reply:
I will update, it's just a flat object.